torch/csrc/jit/mobile/import.cpp

```diff
@@ -222,6 +222,7 @@ class BytecodeDeserializer final {
   // operators from the given model. If it's less than the current runtime,
   // upgrader will be applied at loading stage.
   uint64_t operator_version_;
+  uint64_t model_version_ = 3; // start at 3
```

Comment:
Can this be kMinProducedFileFormatVersion?

Reply:
not related to this change, `bytecode_version` is a better name after we introduce `operator_version`...